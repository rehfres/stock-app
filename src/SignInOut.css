.button {
  display: flex;
  height: 40px;
  width: 178px;
  justify-content: center;
  cursor: pointer;
  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
  transition: .2s ease;
  user-select: none;
  margin: auto auto 16px;
}

@media (min-width: 500px) {
  .button {
    position: absolute;
    width: unset;
    top: 55px;
    right: 30px;
    margin: unset;
  }
}

@media (min-width: 1000px) {
  .button {
    right: 55px;
  }
}

.button:active {
  background: #eee;
  box-shadow: 0 1px 1px 0 rgba(0,0,0,0.14), 0 2px 1px -1px rgba(0,0,0,0.12), 0 1px 3px 0 rgba(0,0,0,0.20);
}

.button__logo {
  padding: 0 8px;
  height: 18px;
  margin: auto 0;
}

.button__logo img {
  height: 18px;
}

@import url('https://fonts.googleapis.com/css?family=Roboto');

.button__text {
  padding: 0 8px 0 16px;
  margin: auto 0;
  font-family: 'Roboto';
  font-size: 14px;
}

.signed-in .button__text {
  padding: 0 16px;
}