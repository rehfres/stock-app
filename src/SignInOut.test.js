import React from 'react';
import SignInOut from './SignInOut';
import {render, fireEvent, cleanup, waitForElement} from 'react-testing-library';

let app;

beforeAll(() => {
  app = render(<SignInOut />);
});

test('renders <SignInOut/>', async () => {
  expect(app).toMatchSnapshot();
});