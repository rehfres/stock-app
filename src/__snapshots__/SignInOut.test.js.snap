// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <SignInOut/> 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="button not-signed-in"
      >
        <div
          class="button__logo"
        >
          <img
            alt="Logo"
            src="g-logo.png"
          />
        </div>
        <p
          class="button__text"
        >
          Sign in with Google
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="button not-signed-in"
    >
      <div
        class="button__logo"
      >
        <img
          alt="Logo"
          src="g-logo.png"
        />
      </div>
      <p
        class="button__text"
      >
        Sign in with Google
      </p>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
